### DISCUSSION TITLE
bb_QC_COG error: "No image files match: /4908980/T1/ref"

### DISCUSSION THREAD

**Discussion by yilewang** [McIntosh-Lab/tvb-ukbb#154] [Help]
Hello! Hope everyone is doing great!

I just have a question about the structural pipeline.. From the `bb_struct_init` log, I found an error about `ref` and `grot_brain_mask`. However, I have no idea what they are doing. I tried to google but I didn't find related information about this `bb_QC_COG` script. Should I worry about this error? Or I can ignore them.
![image](https://user-images.githubusercontent.com/37648360/157376154-b7ff6d43-aa6f-4b36-8784-17e677ec65d6.png)

Thank you!

**Comment by justinliuyuwang** [comment]
Hi Yile! Hope you're doing great too!

You can safely ignore this.
The original UKBB pipeline uses the bb_QC_COG script to produce an QC measure (IDP) based on some reference subject - in this case, 4908990. We have not implemented this part of the pipeline for our own data so it remains untested and unchanged. If you would like to have this QC measure as an output, you would need to point [this variable](https://github.com/McIntosh-Lab/tvb-ukbb/blob/a7c8fd10812e9ba58f73dda0b269c47eb9aa5ff8/bb_structural_pipeline/bb_QC_COG#L26) to a reference subject from your own dataset. Otherwise, you can ignore it.

**Reply by yilewang** [reply]
Thank you for your prompt reply!! I can ignore them without worry lol
